includes/3rd-party/themes/course.php

```diff
@@ -59,7 +59,11 @@ function sensei_load_styles_for_course_theme_variation( $variation_name ) {
 		return;
 	}
 
-	Sensei()->assets->enqueue( 'course-learning-mode-variation', 'css/3rd-party/themes/course/' . $variation_name . '.css', [] );
+	// Styles that are loaded all across when on course theme.
+	Sensei()->assets->enqueue( 'course-theme-styles', 'css/3rd-party/themes/course/style.css' );
```

Comment:
I'm not seeing the CSS in either of these files being applied. The font sizes for the elements are coming from `module.scss`. I was expecting them to be pulled from `course/default.scss` and `course/style.scss` for the Course theme.